RadarSDK/RadarAPIHelper.m

```diff
@@ -67,9 +67,38 @@ - (void)requestWithMethod:(NSString *)method
             }
 
             if (params) {
-                [req setHTTPBody:[NSJSONSerialization dataWithJSONObject:params options:0 error:NULL]];
+                if (!params[@"updatedAtMsDiff"] && !params[@"replays"]) {
```

Comment:
nit: it would be nice if the Android logic matched this `if` check

Reply:
Good callout, fixed: https://github.com/radarlabs/radar-sdk-ios/pull/405/commits/1cd42d92bdca78fd5fa211d06fdce75fc3d2ff55